docs/core/install/runtime.md

```diff
@@ -64,6 +64,9 @@ export PATH=$PATH:$HOME/dotnet
 >
 > Also, add `export DOTNET_ROOT=$HOME/dotnet` to the end of the file.
 
+> [!TIP]
+> This approach lets you install multiple different versions into separate locations and choose explicitly which one to use by which application.
```

Comment:
1. I would remove the tip here and have plain text. This isn't really a tip as it is explaining something.
2. I don't see how this enables choice of explicitly choosing which to use. Maybe you can be more clear such as set the environment variable per app?